flake.nix

```diff
@@ -10,9 +10,6 @@
       };
 
       lib = import ./lib;
-
-      forAllSystems = f: lib.genAttrs lib.systems.flakeExposed (system: f system);
```

Comment:
Let's keep `forAllSystems = lib.genAttrs lib.systems.flakeExposed;` so it can be copy-pasted by newcomers.

Reply:
I do not like unused code in code bases.

Reply:
The point is that `forAllSystems` will get re-used on other attributes by users that copy this repo.

Reply:
Should helper like this not rather than be part of documentation?